cpp/oneapi/dal/algo/knn/common.hpp

```diff
@@ -260,6 +311,16 @@ class descriptor : public detail::descriptor_base<Task> {
     template <typename M = Method, typename = detail::enable_if_brute_force_t<M>>
     auto& set_distance(const distance_t& dist) {
         base_t::set_distance_impl(std::make_shared<detail::distance<distance_t>>(dist));
+        return base_t::get_result_options();
```

Comment:
Why do you need to return result options here?

Reply:
Fixed